src/common/components/table/mempool-table/MempoolTableFilters.tsx

```diff
@@ -0,0 +1,73 @@
+'use client';
+
+import { AddressFilterPopover } from '@/common/components/table/filters/address-filter/AddressFilterPopover';
+import { TransactionTypeFilterPopover } from '@/common/components/table/filters/transaction-type-filter/TransactionTypeFilterPopover';
+import { ClearTxTableFiltersButton } from '@/common/components/table/tx-table/ClearTxTableFiltersButton';
+import { useTxTableFilters } from '@/common/components/table/tx-table/useTxTableFilters';
+import { MODALS } from '@/common/constants/constants';
+import { useAppDispatch } from '@/common/state/hooks';
+import { Button } from '@/ui/Button';
+import { Text } from '@/ui/Text';
+import { Flex, Icon } from '@chakra-ui/react';
+import { Funnel } from '@phosphor-icons/react';
+
+import { openModal } from '../../modals/modal-slice';
+
+const MobileOpenFilterModalButton = () => {
```

Comment:
This seems directly copied from src/common/components/table/filters/TxTableFilters.tsx
Can we make these components reusable?